gremlin-python/src/main/jython/tests/conftest.py

```diff
@@ -65,10 +65,14 @@ def fin():
         request.addfinalizer(fin)
         return client
 
-@pytest.fixture
+@pytest.fixture(params=['v2', 'v3'])
```

Comment:
this is good. doesn't help us with radish test problems though. we still need to do https://issues.apache.org/jira/browse/TINKERPOP-1864 - not sure what's going on there